### DISCUSSION TITLE
Nuxt 3 - Community Presets

### DISCUSSION THREAD

**Discussion by QaahirStewart** [unocss/unocss#1289] [General]
I got my config like this. Trying to add in unocss-preset-scrollbar. Where do I add it in? Tried a few methods but its not building correctly. This is for Nuxt 3. Have it working for Vue but the presets get imported differently.

import { presetScrollbar } from "unocss-preset-scrollbar";

export default defineNuxtConfig({
  buildModules: ["@unocss/nuxt", "@nuxtjs/supabase", "@formkit/nuxt"],
  unocss: {
    attributify: true,
    icons: true,
    components: false,

**Comment by QaahirStewart** [comment]
Found solution, will keep for future reference. Code example.

/unocss.config.ts

import {
  defineConfig,
  presetAttributify,
  presetIcons,
  presetUno,
} from "unocss";
import { presetScrollbar } from "unocss-preset-scrollbar";

export default defineConfig({
  presets: [
    presetScrollbar({
      // config
    }),
    presetUno(),
    presetAttributify(),
    presetIcons(),
  ],
});

**Reply by SnowingFox** [reply]
```ts
import {
  defineConfig,
  presetAttributify,
  presetIcons,
  presetUno,
} from "unocss";
import { presetScrollbar } from "unocss-preset-scrollbar";

export default defineConfig({
  presets: [
      presetScrollbar({
      // config
      }),
      presetUno(),
      presetAttributify(),
      presetIcons(),
  ],
});
```
